docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -17,11 +17,11 @@ menu:
 
 All downloads come with SQLite, MySQL and PostgreSQL support, and are built with
 embedded assets. This can be different for older releases. Choose the file matching
-the destination platform from the [downloads page](https://dl.gitea.io/gitea), copy
+the destination platform from the [downloads page](https://dl.gitea.io/gitea/), copy
 the URL and replace the URL within the commands below:
 
 ```sh
-wget -O gitea https://dl.gitea.io/gitea/1.5.0/gitea-1.5.0-linux-amd64
+wget -O gitea https://dl.gitea.io/gitea/1.7.0/gitea-1.7.0-linux-amd64
```

Comment:
Hmm... I wonder if we should have this pointing to something like "https://dl.gitea.io/gitea/latest/gitea-latest-linux-amd64" and then have that redirect to our latest file.